CMakeLists.txt

```diff
@@ -423,6 +424,25 @@ configure_file (src/glog/logging.h.in glog/logging.h @ONLY)
 configure_file (src/glog/raw_logging.h.in glog/raw_logging.h @ONLY)
 configure_file (src/glog/stl_logging.h.in glog/stl_logging.h @ONLY)
 configure_file (src/glog/vlog_is_on.h.in glog/vlog_is_on.h @ONLY)
+if (WITH_PKGCONFIG)
+  set(VERSION ${PROJECT_VERSION})
```

Comment:
Please use the code style in `CMakeLists.txt` and add space between the command the parenthesis

Reply:
Fixed.

Reply:
Great, thank you!